### PR TITLE
REP-1540 Updated Akka timeout to be based on the configured HTTP Socket timeout.

### PR DESCRIPTION
**Pull request by wdschei**

### CODE REVIEW

File: repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    request = new HttpGet(uri)
+    request.setHeader(HttpHeaders.ACCEPT, MediaType.APPLICATION_XML)
+  }
+
+  after {
+    originServer.stop()
+  }
+
+  val timeouts = List(2000/*, 30000, 45000, 55000, 90000*/)
+  timeouts.foreach { timeout =>
```
Comment (adrianjgeorge) [inline repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala]: I assume you let it run at the larger timeouts and they still passed?

Comment (wdschei) [inline repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala]: Sure did. I just didn't want to leave these in the actual test suite due to it adding over 3 mins to the test.

File: repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import org.mockito.Mockito.when
+import org.openrepose.services.httpclient.{HttpClientResponse, HttpClientService}
+import org.scalatest.junit.JUnitRunner
+import org.scalatest.mock.MockitoSugar
+import org.scalatest.{BeforeAndAfter, FunSpec, Matchers}
+
+import scala.collection.JavaConversions._
+
+@RunWith(classOf[JUnitRunner])
+class AkkaServiceClientImplTestToo extends FunSpec with BeforeAndAfter with Matchers with MockitoSugar with LazyLogging {
```
Comment (adrianjgeorge) [inline repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala]: I'm not a huge fan of creating a second test class like this. how much work would it be to port the old tests?

File: repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala (hunk 1 above)
Comment (adrianjgeorge) [inline repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala]: Figured as much

File: repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala (hunk 2 above)
Comment (wdschei) [inline repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala]: The original Groovy tests don't need the Jetty Server stood up like the new Scala tests do, but just about everything else is in the setUp/before blocks is pertinent to both sets of tests.

Comment (wdschei) [inline repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala]: Should I convert them?

Comment (adrianjgeorge) [inline repose-aggregator/services/service-client/impl/src/test/scala/org/openrepose/services/serviceclient/akka/impl/AkkaServiceClientImplTestToo.scala]: yeah, we should have one test. you can port it either direction.

**Comment by wdschei** [comment]
Retest this please.

**Comment by wdschei** [comment]
Retest this please.

**Pull request merged**